tests/integ/modin/test_apply_snowpark_python_functions.py

```diff
@@ -65,22 +87,27 @@ def test_apply_trunc_default_scale(session):
     assert_frame_equal(s.to_frame().apply(trunc), native_s.to_frame().apply(np.trunc))
 
 
+@pytest.mark.parametrize("scale", [-1, 0, 1, 2])
 @sql_count_checker(query_count=3)
-def test_apply_trunc_scale(session):
+def test_apply_trunc_scale(session, scale):
 
     native_s = native_pd.Series(
-        [-1.0, -0.9, -0.5, -0.2, 0.0, 0.2, 0.5, 0.9, 1.1, 3.14159]
+        [-1.0, -0.9, -0.5234, -0.268, 0.0, 0.23, 0.582, 0.9, 1.1, 3.14159]
```

Comment:
```suggestion
        [-1.0, -0.9, -0.5234, -0.268, 0.0, 0.23, 0.582, 0.9, 1.1, 3.14159, 31.4159, 39, -31.4159, -39]
```

We should add some  2- and 3-digit numbers

Reply:
Added